rdkit/Dbase/DbConnection.py

```diff
@@ -54,19 +50,18 @@ def __init__(self, dbName='', tableName='', user='sysdba', password='masterkey')
     self.cn = None
     self.cursor = None
 
-  def UpdateTableNames(self, dlg):
+  def UpdateTableNames(self, dlg):  # pragma: nocover
```

Comment:
This is a function for a GUI that no longer exists. It would be better to just remove the function.